src/lib/Zikula/Bundle/CoreBundle/Resources/config/translation.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" ?>
```

Comment:
To be honest I have copied this form https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml and just changed translator class to zikula. I do not fully understand what you mean by 'looks confusing' is it it's name or content?